### PR TITLE
[DOC] Test documentation (mfalesni)

### PR DESCRIPTION
**Pull request by mfalesni**
Documenting of my tests or tests that I was touching significantly in recent times.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a6d6e29d2533174265b3ba0ad7109f2810111396:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/automate/test_domain_priority.py:9:1: F401 '_setup_a_provider' imported but unused
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f5e329036b6de564f522dc2a337d603e658f836e:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/automate/test_domain_priority.py:9:1: F401 '_setup_a_provider' imported but unused
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 82a75de974bbbea1e76112566f1dc6d45b9ffa14:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/automate/test_domain_priority.py [hunk 1]
```diff
@@ -176,51 +195,3 @@ def test_priority(
     assert stdout.strip() == original_method_write_data
     ssh_client.run_command("rm -f {}".format(FILE_LOCATION))
     # END OF LAST SIMULATION
-
-
-@pytest.mark.meta(blockers=[1134500])
-def test_override_method_across_domains(
```
Comment (mfalesni) [inline cfme/tests/automate/test_domain_priority.py]: I found out this test became a subset of the previous test, so I dropped it.

File: cfme/tests/cli/test_rake_evm_automate-53.py [hunk 2]
```diff
@@ -11,23 +11,8 @@
...
-        rc, stdout = ssh_client.run_rake_command(
-            "evm:automate:import DOMAIN=Default YAML_FILE=/tmp/Default_backup.yaml PREVIEW=false")
-        Domain(name="Default").update({'enabled': True})
-        if rc != 0:
-            logger.exception("Could not re-improt back the Default domain!: `{}`".format(stdout))
-
-
 @pytest.yield_fixture(scope="function")
-def rake(ssh_client, back_up_default_domain):
+def rake(ssh_client):
```
Comment (mfalesni) [inline cfme/tests/cli/test_rake_evm_automate-53.py]: I did remove the Default domain saving here in accordance with what we have been doing recently.

**Comment by mfalesni** [comment]
Mergy mergy @psav ?

**Pull request merged**